### DISCUSSION TITLE
Is the resume button supposed to be green?

### DISCUSSION THREAD

**Discussion by n-hippie** [witalihirsch/qBitTorrent-fluent-theme#31] [Q&A] [answered]
Using theme version 2.8, fluent-dark, and the resume button is bright green.  Is that by design?  It looks out of place so I'm curious.  I'm running qBittorrent 4.5.0 with Qt6 and Lt2.0.

**Comment by witalihirsch** [comment, marked as answer]
Starting from version 4.5.0, the default icons have been updated, as well as their original names, and I still don’t know how to find out their names to replace them. I don't like it either, I'll try to find a way to change them

**Reply by emwedk** [reply]
Updated - https://github.com/witalihirsch/qBitTorrent-fluent-theme/pull/39
